src/dstack/_internal/core/backends/base/compute.py

```diff
@@ -39,10 +64,20 @@ def run_job(
     ) -> LaunchedInstanceInfo:
         pass
 
+    @abstractmethod
+    def create_instance(
+        self,
+        project: ProjectModel,
+        user: UserModel,
+        instance_offer: InstanceOfferWithAvailability,
+        instance_config: InstanceConfiguration,
+    ) -> LaunchedInstanceInfo:
+        pass
+
```

Comment:
ProjectModel and UserModel cannot be a part of the backend interface. `core` modules should import from `server`.

Reply:
Thanks for the comment! Fixed